migrations/versions/460_add_audit_events_for_g4_and_g5_frameworks.py

```diff
@@ -0,0 +1,29 @@
+"""Add additional indexes for audit_events
+
+Revision ID: 460
+Revises: 450
+Create Date: 2016-01-13 16:45:18.621169
+
+"""
+
+revision = '460'
+down_revision = '450'
+
+from alembic import op
+
+
+def upgrade():
+    # G-Cloud 4
+    op.execute("""
+        INSERT INTO audit_events
+            ("type", "created_at", "user", "data", "object_type", "object_id", "acknowledged")
+        VALUES 
+            ('framework_update', '2015-01-31T14:00:00', 'migration', '{"update": {"status": "expired"}}', 'Framework', 2, FALSE)
```

Comment:
We could even have the user as `migration 460` to make it completely obvious to anyone looking at the event later on.